frontend/ui-pablo/app/components/Organisms/AllBondTable.tsx

```diff
@@ -76,36 +79,47 @@ export const AllBondTable: React.FC = () => {
           </TableRow>
         </TableHead>
         <TableBody>
-          {pools.map((pool, index) => (
+          {allBonds.map((bond, index) => (
             <TableRow
               key={index}
               onClick={handleBondClick}
               sx={{ cursor: "pointer" }}
             >
               <TableCell align="left">
-                {pool.token2 ? (
+                {bond.assetPair.token2 ? (
```

Comment:
@diddt i suggest using bond asset and reward asset instead of token1, token2.

Reply:
This logic is changed to address LP token asset.